docs/software/sciapps/gromacs.md

```diff
@@ -1,4 +1,161 @@
+
+gmx_mpi --version                             # check GROMACS version
+```
+
+The images also provide two alternative views, namely `plumed` and `develop`.
+After starting the pulled image using `uenv start ...` , one may do the following see the available views.
+
+```bash
+$ uenv status
+/user-environment:gromacs-gh200
+  GPU-optimised GROMACS with and without PLUMED, and the toolchain to build your own GROMACS.
+  modules: no modules available
+  views:
+    develop
+    gromacs
+    plumed
+```
+
+The `develop` view has all the required dependencies or GROMACS without the program itself. This is meant for those users who want to use a customized variant of GROMACS for their simulation which they build from source. This view makes it convenient for users as it provides the required compilers (GCC 12) along with the dependencies such as CMake, CUDA, hwloc, Cray MPICH, among many others which their GROMACS can use during build and installation. Users must enable this view each time they want to use their **custom GROMACS installation**.
+
+The `plumed` view contains GROMACS 2022.5 (older version) with PLUMED 2.9.0. This is due to the compatibility requirements of PLUMED. CSCS will periodically update these user environment images to feature newer versions as they are made available.
+
+The `gromacs` view contains GROMACS 2024.1 that has been configured and tested for the highest performance on the Grace-Hopper nodes.
+
+Use `exit` to leave the user environment and return to the original shell.
+
+### How to Run
+
+To start a job, 2 bash scripts are required: a standard SLURM submission script, and a wrapper to start the CUDA MPS daemon (in order to have multiple MPI ranks per GPU).
+
+The CUDA MPS wrapper here: [Oversubscription of GPU cards](https://confluence.cscs.ch/spaces/KB/pages/859015189/Oversubscription+of+GPU+cards)
+
+The wrapper script above needs to be made executable with `chmod +x mps-wrapper.sh`.
+ 
+The SLURM submission script can be adapted from the template below to use the application and the `mps-wrapper.sh` in conjunction.
+
+```bash title="launch.sbatch"
+#!/bin/bash
+
+#SBATCH --job-name="JOB NAME"
+#SBATCH --nodes=1             # number of GH200 nodes with each node having 4 CPU+GPU
+#SBATCH --ntasks-per-node=8   # 8 MPI ranks per node
+#SBATCH --cpus-per-task 32    # 32 OMP threads per MPI rank
+#SBATCH --account=ACCOUNT
+#SBATCH --hint=nomultithread  
+
+export MPICH_GPU_SUPPORT_ENABLED=1
+export FI_CXI_RX_MATCH_MODE=software
+
+export GMX_GPU_DD_COMMS=true
+export GMX_GPU_PME_PP_COMMS=true
+export GMX_FORCE_UPDATE_DEFAULT_GPU=true
+export GMX_ENABLE_DIRECT_GPU_COMM=1
+export GMX_FORCE_GPU_AWARE_MPI=1
+
+srun ./mps-wrapper.sh gmx_mpi mdrun -s input.tpr -ntomp 32 -bonded gpu -nb gpu -pme gpu -pin on -v -noconfout -dlb yes -nstlist 300 -gpu_id 0123 -npme 1 -nsteps 10000 -update gpu
+```
+
+This can be run using `sbatch launch.sbatch` on the login node with the user environment loaded.
```

Comment:
Instead of this, I'd add `--uenv` and `--view` to the submission script, which IMHO is a better practice.

Reply:
Done